### PR TITLE
Removing unnecessary markers from test files

### PR DESCRIPTION
**Pull request by jinengandhi-intel**

### CODE REVIEW

**Review by anjalirai-intel — APPROVED** [review]

File: tests/automation_framework/tests/worker_receipt/test_receipt.py [hunk 1]
```diff
@@ -63,7 +61,6 @@ def test_work_order_create_receipt_success(self):
 
     @pytest.mark.work_order_retrieve_receipt
```
Comment (manojsalunke85) [inline tests/automation_framework/tests/worker_receipt/test_receipt.py]: need to remove work_order_retrieve_receipt marker

**Review by manojsalunke85 — CHANGES_REQUESTED** [review]
Small change to remove the unwanted marker

File: tests/automation_framework/tests/worker_receipt/test_receipt.py (hunk 1 above)
Comment (jinengandhi-intel) [reply tests/automation_framework/tests/worker_receipt/test_receipt.py]: done in the latest commit, had some other receipt API markers, removed those as well.

**Review by manojsalunke85 — APPROVED** [review]
Approved

**Pull request merged**